packages/commonjs/README.md

```diff
@@ -127,6 +127,32 @@ Default: `[]`
 
 Sometimes you have to leave require statements unconverted. Pass an array containing the IDs or an `id => boolean` function. Only use this option if you know what you're doing!
 
+### `dynamicRequireTargets`
+
+Type: `String|Array[String]`<br>
+Default: `[]`
+
+Some modules have dynamic `require` calls in them, or even circular dependencies (which are not handled well by EJS).  
+Including those modules as `dynamicRequireTargets` will simulate a CJS environment for them with support for dynamic and circular dependencies.  
```

Comment:
Please don't use hard link breaks for paragraphs unless two or more lines are needed between them. Backspace so this line wraps naturally. We're big on getting our docs pristine here (even though we have a ways to go yet)